### ISSUE TITLE
Add language identification feature

### ISSUE THREAD

**Issue by rth** [FreeDiscovery/FreeDiscovery#61]
It could be useful to add a language identification to FreeDiscovery.

A possible approach could be to use the Python port of the Google's [language-detection](https://github.com/Mimino666/langdetect)  library, which, as far as I understood, is essentially Naive Bayes classification pre-trained on some wikipedia entries. 

Though the question remains whether it this implementation would scale to ~ 700k  documents.

**Comment by dagr1234** [comment]
yep, there are lots of language ID papers out there but if we can use Google language detection library that might be just fine. 

DG
